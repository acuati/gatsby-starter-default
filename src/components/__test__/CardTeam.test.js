import React from 'react';
import { render } from '@testing-library/react';
import TeamCard from '../teamcard/teamcard';

describe('TeamCard', () => {
  it('renderizamos bien el Componente', () => {
    const { container } = render(
      <TeamCard
        imageSrc="https://via.placeholder.com/150"
        title="Sample Title"
        role= "Jefe de Equipo"
        description="Me gustan las habichuelas"
      />
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
