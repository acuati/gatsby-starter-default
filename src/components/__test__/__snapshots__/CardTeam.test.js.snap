// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TeamCard renderizamos bien el Componente 1`] = `
<div
  class="card-team"
  data-testid="card-team"
>
  <img
    alt="Sample Title"
    class="card-team-image"
    src="https://via.placeholder.com/150"
  />
  <h2
    class="card-team-title"
  >
    Sample Title
  </h2>
  <h2
    class="card-team-role"
  >
    Jefe de Equipo
  </h2>
  <p
    class="card-team-description"
  >
    Me gustan las habichuelas
  </p>
</div>
`;
